.env.template

```
@@ -57,160 +57,133 @@ AUTH_USE_AUTH=false
-
-# Per-level settings (reasoning_level parameter in API)
-# Each level can have its own provider, model, thinking budget, tool iterations, and max output tokens
-# MAX_OUTPUT_TOKENS is optional per level; if not set, uses global DIALECTIC_MAX_OUTPUT_TOKENS
-
-# Minimal level
-# DIALECTIC_LEVELS__minimal__PROVIDER=google
-# DIALECTIC_LEVELS__minimal__MODEL=gemini-2.5-flash-lite
-# DIALECTIC_LEVELS__minimal__THINKING_BUDGET_TOKENS=0
+#
+# Per-level provider, model, and tuning:
+DIALECTIC_LEVELS__minimal__PROVIDER=custom
```

dotenv-linter (4.0.0)

[warning] [LowercaseKey] The DIALECTIC_LEVELS__minimal__PROVIDER key should be in uppercase

(LowercaseKey)